### PR TITLE
MVP for beam player screens

### PR DESCRIPTION
**Pull request by blazsolar**

### CODE REVIEW

File: base/src/main/java/butter/droid/base/torrent/StreamInfo.java [hunk 1]
```diff
@@ -37,7 +38,7 @@
     private String mShowEpisodeTitle;
 
     private Boolean mIsShow = false;
-    private Integer mColor = -1;
+    private int mColor = Color.TRANSPARENT;
```
Comment (aldoborrero) [inline base/src/main/java/butter/droid/base/torrent/StreamInfo.java]: Remove mVariable names

File: mobile/src/main/java/butter/droid/ui/beam/fragment/BeamPlayerFragment.java [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+        seekBar.setSecondaryProgress(0); // hack to make the secondary progress appear on Android 5.0
+        seekBar.setSecondaryProgress(downloadProgress.intValue());
+    }
+
+    @Override
+    public void onStreamStopped() {
+    }
+
+    @Override public void tintProgress(@ColorInt int paletteColor) {
+
```
Comment (aldoborrero) [inline mobile/src/main/java/butter/droid/ui/beam/fragment/BeamPlayerFragment.java]: Remove extra space

File: base/src/main/java/butter/droid/base/torrent/StreamInfo.java (hunk 1 above)
Comment (blazsolar) [inline base/src/main/java/butter/droid/base/torrent/StreamInfo.java]: fixed

File: mobile/src/main/java/butter/droid/ui/beam/fragment/BeamPlayerFragment.java (hunk 2 above)
Comment (blazsolar) [inline mobile/src/main/java/butter/droid/ui/beam/fragment/BeamPlayerFragment.java]: fixed

**Pull request merged**